docker/Dockerfile-Ubuntu-18.04

```diff
@@ -38,6 +38,10 @@ RUN update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/cl
 
 RUN gem install --no-ri --no-rdoc fpm;
 
+RUN useradd -d /home/dronecode -m -s /bin/bash dronecode
+
+USER dronecode
+
 CMD ["/bin/bash"]
 
-WORKDIR "/root/DronecodeSDK"
+WORKDIR "/home/dronecode/DronecodeSDK"
```

Comment:
I guess that should work.

Reply:
Yes it does. You are running as user dronecode inside the container so best to have the workdir in the users home directory rather than roots. Also allows for further enhancement to say mount an application at say /home/dronecode/app and have the sdk and application external to the container.